src/parser/bison_parser.y

```diff
@@ -1070,6 +1086,52 @@ date_literal:
 		}
 	;
 
+interval_literal:
+		int_literal duration_field { $$ = Expr::makeIntervalLiteral($1->ival, $2); delete $1; }
+	|   INTERVAL STRING datetime_field {
+			int duration{0}, chars_parsed{0};
+			// If the whole string is parsed, chars_parsed points to the terminating null byte after the last character
+			if (sscanf($2, "%d%n", &duration, &chars_parsed) != 1 || $2[chars_parsed] != 0) {
+				free($2);
+				yyerror(&yyloc, result, scanner, "Found incorrect interval format. Expected format: INTEGER");
+				YYERROR;
+			}
+			free($2);
+			$$ = Expr::makeIntervalLiteral(duration, $3);
+		}
+	|   INTERVAL STRING {
+			int duration{0}, chars_parsed{0};
+			// 'Seconds' and 'minutes' are the longest accepted interval qualifiers (7 chars) + null byte
```

Comment:
Based on the string comparison cases below, it should be 'seconds' rather than 'Seconds', shouldn't it?